src/cms/static/js/pages/page_order.js

```diff
@@ -58,8 +58,10 @@ function register_event_handlers() {
     u('.drop').each(function(node) {
         u(node).off('dragleave,dragover,drop');
     });
-    // Event handlers for drag events
-    u('.drag').on('dragstart', dragstart);
+    // Event handlers for drag events (delay because of behaviour in Chrome browser)
+    u('.drag').on('dragstart', function(event) {
+        window.setTimeout(dragstart, 50, event);
```

Comment:
```suggestion
        window.setTimeout(dragstart, 0, event);
```

You are using `setTimeout` to schedule the function call after the current execution finished. Waiting for at least 50ms does not make much sense.

Reply:
A ok, I didn't really understand the problem, but just copied from
https://github.com/Integreat/integreat-cms/blob/4b918461b9315001da7f81fa6c165755b90d1b14/src/cms/static/js/tree_drag_and_drop.js#L13
Anyway, I changed both timeouts to `0` now. Thanks for pointing that out!